test/certificate/README.md

```diff
@@ -0,0 +1,48 @@
+# pgBackRest Test Certificates
+
+The certificates in this directory are used for testing purposes only and are not used for actual services.  They are used only by the unit an integration tests and there should be no reason to modify them unless new tests are required.
```

Comment:
Typo: "the unit and integration" should be "the unit and integration"

Reply:
Fixed.